geopandas/testing.py

```diff
@@ -144,7 +144,7 @@ def assert_geoseries_equal(
         Check that both are same type (*and* are GeoSeries). If False,
         will attempt to convert both into GeoSeries.
     check_less_precise : bool, default False
-        If True, use geom_equals_exact. if False, use geom_equals.
+        If True, use geom_equals_exact with relative error of 0.5e-6. if False, use geom_equals.
```

Comment:
```suggestion
        If True, use geom_equals_exact with relative error of 0.5e-6.
        If False, use geom_equals.
```
This shall fix the linting issue.